mautic/mautic
# Fix search by company name

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for 4.0
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
If we coming  from companies list to contacts assigned to company, sometimes show more contacts like we expect. This PR fixed it



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Creat contacts
3. Create two companies `Mautic` and `Mautic inc` and assign one contact to each of them
4. Go to companies list
5. Click on 1 contacts  of `Mautic` company
6. You should see both contact from Mautic and Mautic inc company, but we expect just like one contact
7. After PR works properly
